backend/api/Areas/Project/Models/Search/ProjectFilterModel.cs

```diff
@@ -58,7 +64,7 @@ public ProjectFilterModel(Dictionary<string, Microsoft.Extensions.Primitives.Str
             this.Name = filter.GetStringValue(nameof(this.Name));
             this.StatusId = filter.GetIntNullValue(nameof(this.StatusId));
             this.TierLevelId = filter.GetIntNullValue(nameof(this.TierLevelId));
-
+            this.CreatedByMe = filter.GetValue<bool?>(nameof(this.CreatedByMe));
```

Comment:
does this return `null` I can't remember if it's smart enough.  You might need to create `filter.GetBoolNullValue(...)`